vllm/worker/model_runner.py

```diff
@@ -264,6 +344,65 @@ def __post_init__(self):
             self.lora_index_mapping = [[] for _ in range(self.n_seqs)]
             self.lora_prompt_mapping = [[] for _ in range(self.n_seqs)]
 
+    class InterDataForSeqGroupCache:
```

Comment:
Can we use a common cache class for this as well instead of introducing another similar class?

Reply:
Refactored to a common class